curvefs/test/mds/fs_manager_test.cpp

```diff
@@ -118,23 +117,22 @@ class FSManagerTest : public ::testing::Test {
                                         brpc::SERVER_DOESNT_OWN_SERVICE));
         ASSERT_EQ(0, server_.AddService(&mockCliService2_,
                                         brpc::SERVER_DOESNT_OWN_SERVICE));
-        ASSERT_EQ(0, server_.Start(addr.c_str(), nullptr));
-
-        return;
+        ASSERT_EQ(0, server_.AddService(&fakeCurveFsService_,
+                                        brpc::SERVER_DOESNT_OWN_SERVICE));
+        ASSERT_EQ(0, server_.Start(addr_.c_str(), nullptr));
```

Comment:
Polling port to prevent occupation

Reply:
I use a more agile method - bind to port 0, in this way, kernel will find an unused port for us, https://eklitzke.org/binding-on-port-zero, and `server_.listen_address()` returns the actual IP and port.